bin/install-site.sh

```diff
@@ -143,12 +143,7 @@ EOF
         : # Do nothing on other Ubuntu versions, let's see if we need multiverse
     elif [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"stretch" ]
     then
-        # Install the basic packages we require:
-        cat > /etc/apt/sources.list.d/mysociety-extra.list <<EOF
-# Debian mirror to use, including contrib and non-free:
-deb http://the.earth.li/debian/ stretch main contrib non-free
```

Comment:
I'd like to [move to using `geoipupdate`](https://github.com/mysociety/alaveteli/issues/5550) instead of our custom code, which is [available through `contrib`](https://packages.debian.org/buster/geoipupdate). We [use `geoipupdate` on stretch internally](https://github.com/mysociety/sysadmin/issues/1228#issuecomment-571997221) so I think we want to keep this (and keep going forward to Buster).